### ISSUE TITLE
too much waste

### ISSUE THREAD

**Issue by INFIERNOXIII** [thx114/SuperElementarySchoolUpgradeAsset#3]
Hello first thnx for your mods but there is an issue about waste production each time a truck pass the icon saying waste accumulation appear

**Comment by mwsasser** [comment]
I agree, getting rid of the waste icon seems to be an impossibility.  Can you please fix this?
